### DISCUSSION TITLE
ddetailer & controlnet

### DISCUSSION THREAD

**Discussion by GeekyGhost** [Mikubill/sd-webui-controlnet#649] [Ideas]
I don't know if anyone else has been using ddetailer with controlnet but, controlnet lets you jack it all the way up to 1 on the denoising strength and still keep the exact post of the character and not touch the background. It seems to work well. 

![Screenshot 2023-03-21 112634](https://user-images.githubusercontent.com/111990299/226788539-fc422d73-2502-4495-a1ea-244852026010.png)
![Screenshot 2023-03-21 134910](https://user-images.githubusercontent.com/111990299/226788684-8717fc6f-a36b-40d9-a50c-ac740b88bff3.png)

**Comment by marcsyp** [comment]
curious -- do you have Dreambooth installed as well?  At some point there was a dependency conflict that prevented DB, ddetailer, and controlnet from working together -- has that been resolved?

**Reply by GeekyGhost** [reply]
I learned a while back you could create different venvs for the ones that had conflicts, but then updating meant I had to keep making correction to the code to point DB and etc to the correct venv. Eventually I just set up a different install specifically for training and etc. So I have a Stable Diffusion web-ui and then a training version. I haven't checked to see if the conflict has been resolved as I've gotten used to this setup and found additional benefits.
